docs/voxa-app.rst

```diff
@@ -46,6 +47,79 @@ Voxa Application
       return { tell: 'LaunchIntent.OpenResponse', to: 'die' };
     });
 
+  Also you can use a shorthand version to define a controller. This is very useful when having a controller that only returns a :ref:`transition <transition>`
+
+  .. code-block:: javascript
+
+    voxaApp.onState('launch',
+      {
+        flow: 'yield'
+        reply: 'LaunchIntent.OpenResponse',
+        to: 'nextState'
+      }
+    );
+
+  You can also set the intent that the controller will handle. If set, any other triggered intent will not enter into the controller.
+
+  .. code-block:: javascript
+
+    voxaApp.onState("agreed?", {
+      to: "PurchaseAccepted"
+    }, "YesIntent");
+
+    voxaApp.onState("agreed?", {
+      to: "TransactionCancelled"
+    }, ["NoIntent", "CancelIntent"]);
+
+    voxaApp.onState("agreed?", {
+      to: "agreed?",
+      reply: "Help.ArticleExplanation",
+      flow: "yield"
+    }, "HelpIntent");
+
+    voxaApp.onState("agreed?", {
+      to: "agreed?",
+      reply: "UnknownInput",
+      flow: "yield"
+    });
+
+  **The order on how you declare your controllers matter in Voxa**
+
+  You can set multiple :ref:`controllers <controllers>` for a single state, so how do you know which code will be executed? The first one that Voxa finds. Take this example:
+
+  .. code-block:: javascript
+
+    voxaApp.onState('ProcessUserRequest', (voxaEvent) => {
+      // Some code
+      return { tell: 'ThankYouResponse', to: 'die' };
+    });
+    voxaApp.onState('ProcessUserRequest', (voxaEvent) => {
+      // Some other code
+      return { tell: 'GoodbyeResponse', to: 'die' };
+    });
+
+  If the state machine goes to the `ProcessUserRequest`, the code that will run always will be the first one, so the user will always hear the `ThankYouResponse`.
+
+  The only scenario where this is overrided is when you have more than one for the same state, and one of them has one or more intents defined. If the user triggers the intent that's inside the list of intents of one controller Voxa will give it priority. For example, take this code:
```

Comment:
* overwritten instead of overrided
* more than one .......handler?
* list of one-controller intents, Voxa